cypress/integration/endpoint-tests.js

```diff
@@ -104,7 +104,7 @@ describe(`Visual regression testing for foundation.mozilla.org`, () => {
   });
 
   it(`PNI product page`, function() {
-    cy.visit(`/en/privacynotincluded/products/home-great/`);
+    cy.visit(`/en/privacynotincluded/products/near-reality/`);
```

Comment:
I see you changed this after your comment, but I assume there's another commit to come that manually adds this product to the fake database?

Reply:
yeah, handled in #2897 after which I'll rebase this.